docs/operators/xmlParserOperator.md

```diff
@@ -0,0 +1,83 @@
+Xml Parser
+=============
+
+## Operator Objective
+The XmlParser operator parses XML records and constructs POJOs ("Plain Old Java Objects") from them. The operator also emits each record as a DOM Document if the relevant output port is connected. User can also provide a XSD (XML Schema Definition) to validate incoming XML records. Valid records will be emitted as POJOs / DOM Document while invalid ones are emitted on error port with an error message if the error port is connected.
+
+XmlParser is **idempotent**, **fault-tolerant** and **statically/dynamically partitionable**.
+
+## Class Diagram
+![](images/xmlParser/XmlParser.png)
+## Operator Information
+1. Operator location:**_malhar-library_**
+2. Available since:**_3.2.0_**
+3. Operator state:**_Evolving_**
+4. Java Package:[com.datatorrent.lib.parser.XmlParser](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/parser/XmlParser.java)
+
+## Properties, Attributes and Ports
+### <a name="props"></a>Properties of Xml Parser
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *schemaXSDFile* | [XSD] describing XML data. Incoming records can be validated using the schemaXSDFile. If the data is not as per the requirements specified in schemaXSDFile, they are emitted on the error port.This is an optional property. If the XSD is not provided, incoming tuples are simply converted to POJO or DOM Document without any validations| String | NO | N/A |
```

Comment:
port.This => port. This
converted to POJO or DOM Document => converted to POJOs or DOM Documents